cmake/gcovr.cfg.in

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception.
+
+root = @CMAKE_SOURCE_DIR@
+cobertura = @CMAKE_BINARY_DIR@/coverage/cobertura.xml
+sonarqube = @CMAKE_BINARY_DIR@/coverage/sonarqube.xml
+html-details = @CMAKE_BINARY_DIR@/coverage/coverage.html
+# XXX gcov-executable = @GCOV_EXECUTABLE@
```

Comment:
What is this comment for

Reply:
This is based on an other beman project, but this use presets ...